tools/tools.go

```diff
@@ -0,0 +1,9 @@
+//go:build tools
+
+// This file is used to mark Go modules as used that are only (deps of) tools run during the build.
+package tools
+
+import (
+	// Used by cli/explain/compactgraph/testdata/generate to generate test data.
+	_ "github.com/otiai10/copy"
+)
```

Comment:
I think you already included this dep in `generate.go` so this file should not be needed right?

Otherwise, move this import to `tools/deps/deps.go` as we are keeping our tool deps there.